core-bundle/contao/library/Contao/Image.php

```diff
@@ -81,23 +81,8 @@ public static function getPath($src)
 			return $src;
 		}
 
-		$projectDir = System::getContainer()->getParameter('kernel.project_dir');
-
 		if (str_starts_with($src, 'icon'))
 		{
-			if (pathinfo($src, PATHINFO_EXTENSION) == 'svg')
-			{
-				return 'assets/contao/images/' . $src;
-			}
-
-			$filename = pathinfo($src, PATHINFO_FILENAME);
-
-			// Prefer SVG icons
-			if (file_exists($projectDir . '/assets/contao/images/' . $filename . '.svg'))
-			{
-				return 'assets/contao/images/' . $filename . '.svg';
-			}
```

Comment:
Seems to be unrelated to this PR and breaks BC.

Reply:
As I wrote, this was a Contao 2 BC layer that we forgot to remove. The break would only affect extensions that are compatible with Contao 2 to 5, so effectively none.

Reply:
My extensions still use `.gif` for icons that did not change since Contao 3 🙈
I should fix that anyway but I think the possibility of other extensions having the same issue is quite high.

Reply:
Okay, I guess in this case we have to keep the BC layer. I will adjust the PR accordingly.

Reply:
Did you realize that the code only handles icons from `contao-components/contao` that begin with `icon`? Things like `edit.gif` are not processed anyway.